tfhe/docs/fundamentals/compress.md

```diff
@@ -75,12 +81,14 @@ fn main() {
 
 ```
 
-
 ## Compressed public keys
-This example shows how to compress the classical public keys.
+
+This example shows how to compress the classical public keys:
 
 {% hint style="warning" %}
-It is not currently recommended to use the CompressedPublicKey to encrypt ciphertexts without first decompressing it. In case the resulting PublicKey is too large to fit in memory the encryption with the CompressedPublicKey will be very slow, this is a known problem and will be addressed in future releases.
+It is not currently recommended to use the `CompressedPublicKey` to encrypt ciphertexts without first decompressing them. If the resulting PublicKey is too large to fit in memory, it. may result in significant slowdowns.&#x20;
```

Comment:
what is the `&#x20;` ?

Reply:
I don't know.... never seen it before. 
Gitbook stuff maybe. I'll correct them